server/bleep/src/background.rs

```diff
@@ -175,7 +175,11 @@ impl SyncQueue {
                                 let result = next.run(permit).await;
                                 _ = active.remove(&next.reporef);
 
-                                debug!(?result, "sync finished");
+                                if result.is_ok() {
+                                    debug!(?result, "sync finished");
+                                } else {
+                                    error!(?result, "sync finished");
```

Comment:
A more descriptive error message might be better here?

```suggestion
                                    error!(?result, "sync failed");
```